docs/manual/visualization.md

```diff
@@ -1,7 +1,129 @@
 # Visualization
 
-:::{todo}
+## The `legend-pygeom-vis` viewer
 
-- tips and tricks, even if duplicated (but hard to find) in the Geant4 docs
+For simple geometry visualizatio without particle tracks, it is recommended to
+use `legend-pygeom-vis` from the
+[_legend-pygeom-tools_](https://github.com/legend-exp/legend-pygeom-tools)
+package or the builtin visualization capabilities of your geometry creation
+tool. This viewer also supports adding points from a LH5 file as an overlay over
+the geometry.
 
-:::
+This VTK-based viewer can be run on any system, and _remage_ must not even be
+installed on that device. In our experience, it might be easier to set up than
+the Geant4-based visualization.
+
+## Built-in Geant4 visualization
+
+For more complex visualization, i.e., when you need to visualize the full
+simulated tracks, you have to use the builtin visualization capabilities of
+Geant4. For this it is useful to consult the [upstream visualization docs]. Only
```

Comment:
missing link

Reply:
sorry, this is not direct link but a reference. i just noticed by scrolling down